app/decorators/reviewing_asset.rb

```diff
@@ -5,6 +5,11 @@ def review!
     save
   end
 
+  def unreview!
```

Comment:
Is this method actually necessary for anything?  If we always review/unreview via assignment to attributes, I'd argue for ditching this.